rtsp.go

```diff
@@ -170,9 +180,11 @@ func (rc *rtspCamera) reconnectClient() (err error) {
 		return err
 	}
 
-	codecInfo, err := getStreamInfo(rc.u.String())
-	if err != nil {
-		return err
+	if codecInfo == Unknown {
```

Comment:
Nice, I like how we do not have to modify decoder.go getStreamInfo helper at all, and can just lookup if when using the agnostic model.